src/wrappers/themis/rust/libthemis-sys/bindgen.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+#
+# Generate "src/lib.rs" file with bindings
+#
+# Run this script from "libthemis-sys" directory before release
+# to update raw FFI bindings with newly added functions and types:
+#
+#     ./bindgen.sh
+#
+# You need to have Bindgen, LLVM, rustfmt installed to run this script.
+# Bindgen can be installed with
+#
+#     cargo install bindgen
+#
+# rustfmt can be installed with
+#
+#     rustup component add rustfmt
+#
+# Suitable LLVM can usually be installed from your system's repositories.
+
+set -e
```

Comment:
There are pipes used below, so I'd suggest:
```suggestion
set -e -o pipefail
```

Reply:
> so I'd suggest `-o pipefail`

Thank you, that makes sense. Added.